app/assets/stylesheets/application.scss

```diff
@@ -114,6 +114,10 @@ body {
   font-size: larger;
 }
 
+.non-clickable-list-item {
+  pointer-events: none;
```

Comment:
what's the difference with `cursor: default`?
looks like `mdc-deprecated-list-item` adds `cursor: pointer;`, so it makes me wonder if we should override the same attribute

Reply:
I used `cursor: default`. However, I also added `important!` since material styles are stronger